clients/core-data/client.go

```diff
@@ -22,35 +22,29 @@ import (
 	"fmt"
```

Comment:
How about moving this file to clients/edgex-foundry?

Reply:
The file has been moved.